### PR TITLE
Default cmake install prefix

### LINKED ISSUE #507: Install fail on Windows 8.1
I'm running Visual Studio 2013 Community Edition.

OpenSim (Debug x64) built just fine for me.  However, I got the following error when installing (i.e., right clicking on Install and choosing build):

2>------ Build started: Project: INSTALL, Configuration: Debug x64 ------
2>  -- Install configuration: "Debug"
2>  CMake Error at cmake_install.cmake:31 (file):
2>    file cannot create directory: C:/Program Files/OpenSim/bin.  Maybe need
2>    administrative privileges.

My Windows account is an Administrator account.  In Windows 8.1, being an Administrator is not enough for an application (e.g., Visual Studio) to create directories.  Apparently, the application itself needs to be given Administrative privileges. 

I got passed the above problem in two ways (the 2nd being better).
1.  I created all the needed directories myself (tedious).
2.  I started Visual Studio by right clicking Visual Studio and choosing "Run as Administrator".  This means you can't start Visual Studio by double clicking the .sln file, however.  You have to start Visual Studio using "Run as Administrator" and then navigate to .sln file.

There is probably a third way... Setting Visual Studio to always run with Administrator privileges.  I haven't tried that. 

Not sure there is something that could be done on the OpenSim side.  However, some additions to the GitHub readme where it gives directions for Installing might help some users get past this minor hitch. 

(By the way, I encountered the same error when installing Simbody.  So, this issue surely has to do with how I installed Visual Studio 2013 Community Edition.)

### PR DESCRIPTION
**Pull request by chrisdembia**
Fixes #507 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ On Mac using Xcode
    changing the `CMAKE_INSTALL_PREFIX` variable. We'll assume you set it to
    `~/opensim-core`. If you choose a different installation location, make
    sure to use *yours* where we use `~/opensim-core` below. You should *not*
-   use `/usr/`, `/usr/local/` (because our installation does not yet conform to the [FHS](http://www.pathname.com/fhs/)).
+   use `/usr/`, `/usr/local/`, etc (because our installation does not yet conform to the [FHS](http://www.pathname.com/fhs/)).
```
Comment (sherm1) [inline README.md]: Needs a "." after "etc".

**Comment by sherm1** [comment]
This looks good to me.

**Comment by chrisdembia** [comment]
@fcanderson, @aymanhab what do you think?

**Comment by chrisdembia** [comment]
@sherm1 would you like a similar change in Simbody?

**Comment by sherm1** [comment]
> @sherm1 would you like a similar change in Simbody?

No, thanks Chris. I'm happy with the default installation locations for Simbody, which are the standard ones on all platforms, like locale-correct `Program Files` or `Program Files (x86)` on Windows, /usr/local on Debian-compatible, etc. Those aren't easy to figure out by hand so I like having CMake set them correctly. A user can easily change to some other location if they don't have admin or root permission.

**Comment by chrisdembia** [comment]
I'll merge this soon unless anyone objects.

**Pull request merged**